bluesky_kafka/logging_utils.py

```diff
@@ -0,0 +1,129 @@
+# See https://docs.confluent.io/platform/current/installation/configuration/producer-configs.html and
+#     https://docs.confluent.io/platform/current/installation/configuration/consumer-configs.html
+CONFIG_KEY_WHITELIST = {
+    ##
+    # -- Common Keys
+    "bootstrap.servers",
+    "client.dns.lookup",
+    "client.id",
+    "connections.max.idle.ms",
+    "receive.buffer.bytes",
+    "request.timeout.ms",
+    # "sasl.client.callback.handler.class",
+    # "sasl.jaas.config",
+    # "sasl.kerberos.service.name",
+    # "sasl.login.callback.handler.class",
+    # "sasl.login.class",
+    # "sasl.mechanism",
+    # "sasl.oauthbearer.jwks.endpoint.url",
+    # "sasl.oauthbearer.token.endpoint.url",
+    "security.protocol",
+    "send.buffer.bytes",
+    "socket.connection.setup.timeout.max.ms",
+    "socket.connection.setup.timeout.ms",
+    # "ssl.enabled.protocols",
+    # "ssl.keystore.type",
+    # "ssl.protocol",
+    # "ssl.provider",
+    # "ssl.truststore.type",
+    "auto.include.jmx.reporter",
+    "enable.metrics.push",
+    "interceptor.classes",
+    "metadata.max.age.ms",
+    "metric.reporters",
+    "metrics.num.samples",
+    "metrics.recording.level",
+    "metrics.sample.window.ms",
+    "reconnect.backoff.max.ms",
+    "reconnect.backoff.ms",
+    "retry.backoff.max.ms",
+    "retry.backoff.ms",
+    # "sasl.kerberos.kinit.cmd",
+    # "sasl.kerberos.min.time.before.relogin",
+    # "sasl.kerberos.ticket.renew.jitter",
+    # "sasl.kerberos.ticket.renew.window.factor",
+    # "sasl.login.connect.timeout.ms",
+    # "sasl.login.read.timeout.ms",
+    # "sasl.login.refresh.buffer.seconds",
+    # "sasl.login.refresh.min.period.seconds",
+    # "sasl.login.refresh.window.factor",
+    # "sasl.login.refresh.window.jitter",
+    # "sasl.login.retry.backoff.max.ms",
+    # "sasl.login.retry.backoff.ms",
```

Comment:
Looking at this again, I may have gone a bit overboard with the redacting I guess. I was just erring on the side of caution in case one of these SASL parameters would leak something sensitive. Happy to just redact a smaller subset like `sasl.password`